contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -319,6 +420,7 @@ contract StakingVault is IStakingVault, OwnableUpgradeable {
         ERC7201Storage storage $ = _getStorage();
```

Comment:
let's add another fn `resetLock` that can be called `onlyOwner` and ONLY WHEN VaultHub is detached

Reply:
done